### PR TITLE
Add trigger column; update column order

### PR DESCRIPTION
**Pull request by goodoldneon**
## Description
- Add trigger column
- Update column order
- Change default Dev Server column visibility

## Type of change (choose one)
- [ ] Chore (refactors, upgrades, etc.)
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Security fix (non-breaking change that fixes a potential vulnerability)
- [ ] Docs
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist
- [ ] I've linked any associated issues to this PR.
- [ ] I've tested my own changes.

*[Check our Pull Request Guidelines](https://github.com/inngest/inngest/blob/main/docs/PULL_REQUEST_GUIDELINES.md)*

### CODE REVIEW

File: ui/packages/components/src/RunsPage/RunsPage.tsx [hunk 1]
```diff
@@ -212,10 +226,20 @@ export function RunsPage({
...
-  const options: Option[] = columns.map((column) => ({
-    id: column.accessorKey,
-    name: column.header?.toString() || column.accessorKey,
-  }));
+  const options = useMemo(() => {
+    const out = [];
+    for (const column of columns) {
+      if (!isColumnID(column.id)) {
+        continue;
+      }
```
Comment (djfarrelly) [inline ui/packages/components/src/RunsPage/RunsPage.tsx:234]: **question (non-blocking**: I'm guessing this logic is just for type safety? Part of TS that I don't love 😆

File: ui/packages/components/src/RunsPage/columns.tsx [hunk 2]
```diff
@@ -33,6 +56,52 @@ const columns = [
...
+  }),
+  columnHelper.display({
+    cell: (props) => {
+      const data = props.row.original;
+
+      if (data.isBatch) {
+        return 'Batch';
+      }
+      if (data.cronSchedule) {
+        return data.cronSchedule;
```
Comment (djfarrelly) [inline ui/packages/components/src/RunsPage/columns.tsx:69]: **design**: Design wise, we should make at least crons monospace. We can use IDCell for now, but ideally that should be more a more generic component name.

**Review by djfarrelly — APPROVED** [review]
Approving, but with one requested change for monospace cron schedules

File: ui/packages/components/src/RunsPage/RunsPage.tsx (hunk 1 above)
Comment (goodoldneon) [reply ui/packages/components/src/RunsPage/RunsPage.tsx:234]: TypeScript is warning about a real thing here. TanStack Table columns aren't required to have IDs. We _should_ always specify them, but technically they're optional

Comment (djfarrelly) [reply ui/packages/components/src/RunsPage/RunsPage.tsx:234]: Thanks - makes a lot of sense!

**Review by djfarrelly — APPROVED** [review]

**Pull request merged**